### PR TITLE
Add azure_sql and vnet deployment files

### LINKED ISSUE #5: Creating database, VNet, service endpoint connection using bicep

### PR DESCRIPTION
**Pull request by raqeebr**
Add bicep files for;
- Azure SQL
- VNet with Service Endpoints
- App service integration

Updated App service SKU to `B1` as vnet integration is not supported in Free tier

### CODE REVIEW

File: azure-app-service-vnet-template/Infrastructure/azure_sql.bicep [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+param sku_name string
+param sku_tier string
+param vnet_subnet_id string
+param tags object
+
+resource sql_server 'Microsoft.Sql/servers@2022-02-01-preview' = {
+  name: sql_server_name
+  location: location
+  properties: {
+    administratorLogin: 'sqlsrvdbadmin'
```
Comment (KamalRathnayake) [inline azure-app-service-vnet-template/Infrastructure/azure_sql.bicep]: make the username and password parameterized

File: azure-app-service-vnet-template/Infrastructure/main.bicep [hunk 2]
```diff
@@ -1,42 +1,66 @@
-targetScope='subscription'
+targetScope = 'subscription'
 
 param resource_group_name string = 'webapp-templates'
 param location string = 'eastasia'
 param app_service_plan_name string = 'webapp-template-app-plan'
 param app_service_name string = 'webapp-template-app'
-param sku string = 'F1'
```
Comment (KamalRathnayake) [inline azure-app-service-vnet-template/Infrastructure/main.bicep:7]: pass the parameters into the module

File: azure-app-service-vnet-template/Infrastructure/vnet.bicep [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+resource vnet 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+  name: vnet_name
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: vnet_addr_prefixes
+    }
+    subnets: [
+      {
+        name: 'todoappsubnet01'
```
Comment (KamalRathnayake) [inline azure-app-service-vnet-template/Infrastructure/vnet.bicep]: rename the net and subnet according to the architecture diagram

File: azure-app-service-vnet-template/Infrastructure/azure_sql.bicep (hunk 1 above)
Comment (raqeebr) [reply azure-app-service-vnet-template/Infrastructure/azure_sql.bicep]: Resolved

File: azure-app-service-vnet-template/Infrastructure/main.bicep (hunk 2 above)
Comment (raqeebr) [reply azure-app-service-vnet-template/Infrastructure/main.bicep:7]: Resolved

File: azure-app-service-vnet-template/Infrastructure/vnet.bicep (hunk 3 above)
Comment (raqeebr) [reply azure-app-service-vnet-template/Infrastructure/vnet.bicep]: Resolved

**Pull request merged**